package.json

```diff
@@ -12,6 +12,6 @@
     "jest": "^27.1.0"
   },
   "scripts": {
-    "test": "jest"
+    "test": "jest sign-up"
```

Comment:
The settings in the `package.json` should be targeted to a folder-`test` rather than just one file.